bulbs/utils/methods.py

```diff
@@ -99,3 +101,9 @@ def get_overridable_template_name(parent_name, child_name):
         template_name = parent_name
 
     return template_name
+
+def get_video_object(video_id):
+    video_model_name = getattr(settings, "VIDEO_MODEL", "")
+    video_model = apps.get_model(video_model_name)
+    video = get_object_or_404(video_model, videohub_ref_id=int(video_id))
```

Comment:
probably should just return None instead of 404ing, since this would be used outside of views.